guides/1.4/routing.md

```diff
@@ -590,7 +590,7 @@ review_path  DELETE  /reviews/:id       HelloWeb.ReviewController :delete
   user_path  DELETE  /users/:id         HelloWeb.UserController :delete
 ```
 
-Although technically scopes can also be nested (just like resources), the use of nested scopes is generally discouraged because it can sometimes make our code confusing and less clear. With that said, suppose that we had a versioned API with resources defined for images, reviews and users. Then technically we could  setup routes for the versioned API like this:
+技術的にはスコープは入れ子にすることもできますが（リソースと同じように）、入れ子にしたスコープを使用することは一般的にはオススメできません。とはいえ、画像、レビュー、ユーザー用のリソースを定義したバージョン管理されたAPIがあったとしましょう。その場合、技術的には以下のようにバージョニングされたAPIのルートを設定できます。
```

Comment:
`because it can sometimes make our code confusing and less clear`
の訳をいれてみました。オススメできないことが伝えたいところですので、いまのまま省略はありだとおもいます。

```suggestion
技術的にはスコープは入れ子にすることもできますが（リソースと同じように）、入れ子にしたスコープを使用することはコードを紛らわしく複雑にすることがあるので、一般的にはオススメできません。とはいえ、画像、レビュー、ユーザー用のリソースを定義したバージョン管理されたAPIがあったとしましょう。その場合、技術的には以下のようにバージョニングされたAPIのルートを設定できます。
```